core/src/test/resources/sql/misc.iq

```diff
@@ -2301,17 +2301,6 @@ where false;
 
 !ok
 
-# [CALCITE-2447] POWER, ATAN2 functions fail with NoSuchMethodException
```

Comment:
why delete this test?

Reply:
The `POWER` function is no longer available in `SqlStdOperatorTable`. Is there any way of specifying a library to use here?

Reply:
I don't think there is, but the question is whether there should be a standard power function.